src/Compatibility/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/ApiCompatServiceProvider.cs

```diff
@@ -17,7 +17,7 @@ internal sealed class ApiCompatServiceProvider
         private readonly Lazy<ISuppressibleLog> _compatibilityLogger;
         private readonly Lazy<IApiCompatRunner> _apiCompatRunner;
 
-        internal ApiCompatServiceProvider(Func<ISuppressionEngine, ISuppressibleLog> logFactory,
+        public ApiCompatServiceProvider(Func<ISuppressionEngine, ISuppressibleLog> logFactory,
```

Comment:
Why did you decide to make this `public`? I don't see new invocations.

Reply:
The class is already internal and I prefer to have public API in an internal type over internal API in an internal type.